meetings/tool/src/main/java/org/sakaiproject/meetings/controller/MeetingsController.java

```diff
@@ -593,6 +608,32 @@ public void deleteMeeting(@PathVariable String meetingId) throws MeetingsExcepti
 			throw new MeetingsException(e.getLocalizedMessage());
 		}
 	}
+
+	@GetMapping(value = "/meeting/{meetingId}/attendanceReport", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<?> getMeetingAttendanceReport(@PathVariable String meetingId, @RequestParam(required = false) String format) throws MeetingsException {
+		checkCurrentUserInMeeting(meetingId);
+		Meeting meeting = meetingService.getMeeting(meetingId);
+		String onlineMeetingId = meetingService.getMeetingProperty(meeting, ONLINE_MEETING_ID);
+		String organizerEmail = meetingService.getMeetingProperty(meeting, ORGANIZER_USER);
+		checkUpdatePermissions(meeting.getSiteId());
+		microsoftCommonService.inicializeMeetingNameColumns(rb.getString("meeting.attendance_report"), rb.getString("meeting.column_name"), rb.getString("meeting.column_email"), rb.getString("meeting.column_role"), rb.getString("meeting.column_duration"), rb.getString("meeting.interval_duration"), rb.getString("meeting.entry_date"), rb.getString("meeting.exit_date"), rb.getString("meeting.details"));
+
+		try {
+			List<AttendanceRecord> attendanceRecords = microsoftCommonService.getMeetingAttendanceReport(onlineMeetingId, organizerEmail);
+			 if ("csv".equalsIgnoreCase(format)) {
```

Comment:
Please use a constant defined at the top, something like REPORT_FORMAT_CSV = "csv"